packages/cli/src/util/create-git-meta.ts

```diff
@@ -54,6 +54,14 @@ export async function createGitMeta(
   const dirty = dirtyResult.value;
   const commit = commitResult.value;
 
+  let ciMetadata: CiMetadata = {};
+  if (process.env.GITHUB_ACTIONS && process.env.GITHUB_ACTOR) {
```

Comment:
https://docs.github.com/en/actions/reference/variables-reference

Name | Description
-- | --
`GITHUB_ACTIONS` | Always set to `true` when GitHub Actions is running the workflow. You can use this variable to differentiate when tests are being run locally or by GitHub Actions.
`GITHUB_ACTOR` | The name of the person or app that initiated the workflow. For example, `octocat`.